src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeMultiLoadRunner.java

```diff
@@ -0,0 +1,68 @@
+package org.jsmart.zerocode.core.runner.parallel;
+
+import org.jsmart.zerocode.core.domain.TestMapping;
+import org.jsmart.zerocode.core.domain.TestMappings;
+import org.junit.runner.Description;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Request;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.ParentRunner;
+import org.junit.runners.model.InitializationError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ZeroCodeMultiLoadRunner extends ParentRunner<TestMapping> {
```

Comment:
No usage?

Reply:
still WIP-, but the usage is not yet defined,will raise an issue and add to tech-debt or backlog/enhancement if needed.